swig/CMakeLists.txt

```diff
@@ -44,7 +44,7 @@ foreach(incdir ${GNURADIO_RUNTIME_INCLUDE_DIRS})
     list(APPEND GR_SWIG_INCLUDE_DIRS ${incdir}/gnuradio/swig)
 endforeach(incdir)
 
-set(GR_SWIG_LIBRARIES gnuradio-stellarstation ${CONAN_LIBS})
+set(GR_SWIG_LIBRARIES gnuradio-stellarstation)
```

Comment:
Works either way for me, so let me know if it fails and I'll bring it back. It might be finding my globally installed libs at this step (?), but I'm not really sure.